.github/workflows/triage_issues.yml

```diff
@@ -0,0 +1,31 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Triage new issues for @chakra-ui/vue
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  issues:
+    types: [opened, reopened]
+    
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "triage"
+  triage:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - name: Add issue to Triage and Backlog project board
+        uses: actions/github-script@v1
+        env: 
+          COLUMN_ID: 9339326
+        with:
+          gihub-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this supposed to be `github-token`?

Reply:
Yeah it is.

Reply:
The workflow is triggered on the opening or reopening of an issue. It will then create a new card in this https://github.com/chakra-ui/chakra-ui-vue/projects/3 in the column triage column(that is what the id is for). As for the secrets, you can see it here https://help.github.com/en/actions/configuring-and-managing-workflows/authenticating-with-the-github_token